packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -289,6 +343,52 @@ export class WorkspaceCommandContribution implements CommandContribution {
         }
         return parentUri.resolve(base);
     }
+
+    private async openWorkspaceRootConfig(): Promise<void> {
+        const rootConfigUri = this.workspaceService.getWorkspaceConfigFileUri();
```

Comment:
I don't like exposing an existence of the config file, I see that a single external client to `getWorkspaceConfigFileUri` and `createWorkspaceConfigFile` is a command to open it. Maybe we should consider dropping a command since it is not very useful actually. It should allow to remove `createWorkspaceConfigFile` and make `getWorkspaceConfigFileUri` protected.

Reply:
ok i will remove it